package.json

```diff
@@ -37,8 +37,8 @@
 		"url": "https://github.com/pact-foundation/pact-node/issues"
 	},
 	"dependencies": {
-		"@pact-foundation/pact-mock-service": "~2.1.0",
-		"@pact-foundation/pact-provider-verifier": "~1.1.1",
+		"@pact-foundation/pact-mock-service": "^2.1.0",
```

Comment:
Also, build is green except for one mac osx build that times out. Need to get to the source of flakiness - could just be slowness starting the Ruby process.